packages/react-tweet/src/twitter-theme/tweet-info-created-at.tsx

```diff
@@ -1,9 +1,9 @@
 'use client'
```

Comment:
I've been thinking this could probably no longer be a `client` component.

We're using a unique format for the date, and the tweet date will not change. 🤔 

Open to discuss about it.

I created a custom server component for my website, implementing these changes so `date-fns` was not bundled, and it works just fine.

https://github.com/jahirfiquitiva/jahir.dev/blob/main/src/components/ui/blog/mdx/tweet/tweet-created-at.tsx

Reply:
This was initially `use client` because the previous version of the X embedded tweet had relative times (like 5 days ago) and calculating the relative time in the server would end up in a cached and incorrect result.

Due to how it is today, it's most likely safe to make a server component and it's actually better to avoid running the logic behind the formatter in the browser.

Reply:
I just saw that there's a comment in the component that talks exactly about this so yeah, it's better to make it a server component